### DISCUSSION TITLE
wallet connect qr code photo not showing

### DISCUSSION THREAD

**Discussion by Rickshanelorie** [WalletConnect/walletconnect-monorepo#3440] [Developer Support]
quick fix for wallet connect qr code photo not showing ???

**Comment by Rickshanelorie** [comment]
![walletconnect](https://github.com/WalletConnect/walletconnect-monorepo/assets/132715920/566d158a-b9e2-4f24-954f-026500d503bf)

**Reply by Rickshanelorie** [reply]
@glitch-txs sir what's the fix?

**Reply by Rickshanelorie** [reply]
@glitch-txs

**Reply by glitch-txs** [reply]
Can you share a [MRE](https://en.wikipedia.org/wiki/Minimal_reproducible_example#:~:text=In%20computing%2C%20a%20minimal%20reproducible,to%20be%20demonstrated%20and%20reproduced.)?

**Reply by Rickshanelorie** [reply]
@glitch-txs "import {
EthereumClient,
w3mConnectors,
w3mProvider,
WagmiCore,
WagmiCoreChains,
WagmiCoreConnectors,
} from "https://unpkg.com/@web3modal/ethereum@2.6.2";
import { Web3Modal } from "https://unpkg.com/@web3modal/html@2.6.2";

// 0. Import wagmi dependencies
const { bscTestnet, bsc } = WagmiCoreChains;
const { configureChains, createConfig } = WagmiCore;

// 1. Define chains
const chains = [bscTestnet, bsc];
const projectId = "#";

// 2. Configure wagmi client
const { publicClient } = configureChains(chains, [w3mProvider({ projectId })]);
const wagmiConfig = createConfig({
autoConnect: true,
connectors: [
...w3mConnectors({ chains, version: 2, projectId }),
new WagmiCoreConnectors.CoinbaseWalletConnector({
chains,
options: {
appName: "#",
},
}),
],
publicClient,
});

// 3. Create ethereum and modal clients
const ethereumClient = new EthereumClient(wagmiConfig, chains);
export const web3Modal = new Web3Modal(
{
projectId,
themeVariables: {
"--w3m-accent-color":
"background: linear-gradient(157.81deg, #DEF9FA -43.27%, #BEF3F5 -21.24%, #9DEDF0 12.19%, #7DE7EB 29.82%, #5CE1E6 51.94%, #33BBCF 90.29%);",
},
walletImages: {
safe:
"https://pbs.twimg.com/profile_images/1566773491764023297/IvmCdGnM_400x400.jpg",
},
},
ethereumClient
);

**Reply by glitch-txs** [reply]
Try updating the packages to 2.7.1 check
https://github.com/WalletConnect/web3modal-examples/tree/main/web3modal-wagmi-html-cdn

Also here are the available themes:
https://docs.walletconnect.com/2.0/advanced/walletconnectmodal/theming
